walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -19,24 +19,84 @@
 import java.util.Collection;
 import java.util.Collections;
 import java.util.List;
+import java.util.stream.Collectors;
 
-/**
- * Given a list of events, returns a list of possible time ranges for the meeting request.
- */
 public final class FindMeetingQuery {
+
+  /** Add time range to possible time slots if there is enough time between firstTime and secondTime. */
+  private void addTimeRangeIfAppropriate(Collection<TimeRange> possibleTimes, long duration, 
+    int firstTime, int secondTime, boolean inclusive) {
+    if ((secondTime - firstTime) >= duration) {
+      possibleTimes.add(TimeRange.fromStartEnd(firstTime, secondTime, inclusive));
+    }
+  }
+
+  /** Returns a list of time slots that work for mandatory attendees. */
+  private List<TimeRange> handleMandatoryAvailability(List<TimeRange> eventTimes, long duration) {
```

Comment:
handleMandatoryAvailability doesn't tell much info. rename the function to be more clear.